### ISSUE TITLE
OCPP SecurityEventNotification Not Forwarding to CentralSystemMessageHandler

### ISSUE THREAD

**Issue by couryrr-afs** [citrineos/citrineos-core#19]
I am running a charge point (cp) that is connecting to CitrineOS locally. I can see the cp making a connection on the WebSocket and a successful exchange of the BootNotification message both in CitrineOS and the cp. The cp was then giving a timeout error. Looking into the logs I see that a SecurityEventNotification is being sent from the cp. In CitrineOS the message is received in RabbitMqSender but I am not seeing this go to CentralSystemImpl (like the BootNotification).

```
2023-11-13 17:42:58.349	INFO	/src/server/server.ts:357	CitrineOS Logger:CentralSystemImpl	Successfully connected new charging station. cp001
2023-11-13 17:42:58.351	DEBUG	/node_modules/@citrineos/util/lib/queue/rabbit-mq/sender.js:119	CitrineOS Logger:RabbitMqSender	Publishing to citrineos: {
  origin: 'cs',
  eventGroup: 'general',
  action: 'SecurityEventNotification',
  state: 1,
  context: {
    correlationId: 'e946d2c9-86d8-4770-ba61-d12668027125',
    stationId: 'cp001',
    tenantId: ''
  },
  payload: {
    techInfo: 'Charging Station powered up! Firmware version: ',
    timestamp: '2023-11-09T19:21:58.706Z',
    type: 'StartupOfTheDevice'
  }
}
```

I believe that the structure is correct and the message name is proper from the cp based on the OCPP spec.

**Comment by thanaParis** [comment]
Hi @couryrr-afs !
Unfortunately, there is not currently a handler for the SecurityEventNotification action, so the message has nowhere to go. One will be added to the next rc branch, in order to meet Core certification profile requirements. That branch should be available within a week or two, though if you feel you know what's needed to handle SecurityEventNotifications it would be great if you could open your own pr!

**Comment by couryrr-afs** [comment]
Looking forward to the next release. Closing issue with the expected update.

**Comment by drmrd** [comment]
@thanaParis: Has `SecurityEventNotification` handling been added to one of the more recent releases?

**Comment by thanaParis** [comment]
@drmrd not as of yet. There's been some delays in my schedule, apologies. I will ping this thread when the handling is available.

**Comment by drmrd** [comment]
Thanks, @thanaParis. I'll keep an eye out.

If the changes are already available on a branch, we'd love to test them out.

**Comment by drmrd** [comment]
@couryrr-afs: Are you able to re-open this issue?

**Comment by joll90** [comment]
@drmrd @couryrr-afs merged into RC from PR #28
